hardhat.config.ts

```diff
@@ -111,7 +111,26 @@ const hardhatConfig: HardhatUserConfig = {
     apiKey: {
       optimisticEthereum: OPTIMISTIC_ETHERSCAN_KEY,
       arbitrumOne: ARBISCAN_KEY,
+      "optimism-testnet": OPTIMISTIC_ETHERSCAN_KEY,
     },
+    customChains: [
+        {
+          network: 'sepolia',
+          chainId: 11155111,
+          urls: {
+            apiURL: 'https://api-sepolia.etherscan.io/api',
+            browserURL: 'https://sepolia.etherscan.io',
+          },
+        },
+        {
+            network: 'optimism-testnet',
```

Comment:
```suggestion
            network: 'optimism-sepolia',
```

Reply:
I made it "optimisticSepolia" to be in sync with apiKey